### PR TITLE
Boilr convert

### PR DESCRIPTION
**Pull request by justAkshitAgrawal**

### CODE REVIEW

File: .drone.yml.multiarch [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# if you want to publish a multi-architecture image, replace the
```
Comment (rajatharanganath) [inline .drone.yml.multiarch:1]: We can remove this

File: README.md [hunk 2]
```diff
@@ -46,13 +48,12 @@ The plugin is available for the following architectures:
...
+        image: harnesscommunity/drone-gcp-oidc:latest
         settings:
-                project_number: 22819301
+                project_id: 22819301
                 pool_id: d8291ka22
-                pool_id: kda91fa
-                service_account_email: test-gcp@harness.io
-                service_account: svr-account1
+                service_account_email_id: test-gcp@harness.io
+                provider_id: svr-account1
```
[the remark this answers is not in the document]
Comment (justAkshitAgrawal) [reply README.md:56]: @rajatharanganath aren't these settings needed for the plugin? Won't removing them from example cause confusion?

File: .drone.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+kind: pipeline
```
[the remark this answers is not in the document]
Comment (justAkshitAgrawal) [reply .drone.yml]: Removed

File: .drone.yml.multiarch (hunk 1 above)
Comment (justAkshitAgrawal) [reply .drone.yml.multiarch]: Removed

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rajatharanganath) [reply README.md:56]: Got it, my bad!

**Review by rajatharanganath — APPROVED** [review]

**Pull request merged**